features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -478,6 +555,244 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
             });
     };
 
+    const resolveAttributeSupportedByDefaultRow = (): ReactElement => {
+        const isSupportedByDefaultCheckboxDisabled: boolean = !hasMapping
+            || (
+                accountVerificationEnabled
+                && selfRegistrationEnabled
+                && claim?.claimURI ===
+                    ClaimManagementConstants.EMAIL_CLAIM_URI
+                && usernameConfig?.enableValidator === "true"
+            );
+
+        return (
+            <TableRow>
+                <TableCell>
+                    { t("claims:local.forms.profiles.displayByDefault") }
+                    <Tooltip
+                        compact
+                        trigger={ (
+                            <Icon name="info circle" color="grey" className="ml-1" />
+                        ) }
+                        content={
+                            t("claims:local.forms.profiles.displayByDefaultHint")
+                        }
+                    />
+                </TableCell>
+                <TableCell align="center">
+                    <Field.Checkbox
+                        ariaLabel="Display by default in console"
+                        name="consoleSupportedByDefault"
+                        required={ false }
+                        defaultValue={ claim?.profiles?.console?.supportedByDefault ?? claim?.supportedByDefault }
+                        data-testid={
```

Comment:
Shall we use data component id for new elements? Check the other places as well.

Reply:
Fixed with bc58d4c77746d8ad71c373b64ab0ce547c7136ac